spec/precompiler.js

```diff
@@ -307,7 +307,7 @@ describe('precompiler', function() {
       Precompiler.loadTemplates(
         { files: [__dirname + '/artifacts'], extension: 'hbs' },
         function(err, opts) {
-          equal(opts.templates.length, 2);
+          equal(opts.templates.length, 1);
```

Comment:
As `non.default.extension.amd.hbs` artifact was removed in this PR, the count is now back to 1.